modules/localstore.js

```diff
@@ -0,0 +1,24 @@
+export default class Storage {
+  static setBook(books) {
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static addBook(book) {
+    // Check if books exist then add them to Local Storage
+    const books = JSON.parse(localStorage.getItem('books')) || [];
+    books.push(book);
+    this.setBook(books);
+  }
+
+  static getBooks() {
```

Comment:
Kindly use the ES6 arrow function syntax instead of the function declaration where ever possible. This is one of the requirements for this milestone. ⬇️
![image](https://user-images.githubusercontent.com/33268927/213628825-94c7fa25-dd48-4f37-a071-24e1adbb809e.png)